### ISSUE TITLE
Error with googlenet from modelstore

### ISSUE THREAD

**Issue by WilliamJuel** [NVIDIA/DIGITS#1629]
I have been working a bit with finetuning the alexnet and googlenet from the model store. Fintuning the pretrained alexnet works great for me with no trouble but I cannot get the googlenet model to work. I have changed the name of the innerproduct layers and num_output to predict my number of classes. I also tried to not even include the model but only use the prototxt and I get this error in both cases:

ERROR: Check failed: axis_index < num_axes() (2 vs. 2) axis 2 out of range for 2-D Blob with shape 64 1024 (65536)

DIGITS 5.0
caffe 0.16 
[caffe_output.txt](https://github.com/NVIDIA/DIGITS/files/981553/caffe_output.txt)

Can anyone give me a suggestion to what's causing this issue. 

Thanks a lot

**Comment by lukeyeager** [comment]
@drnikolaev possibly related to caffe 0.16?

**Comment by WilliamJuel** [comment]
@lukeyeager  So, I made the googlenet from the model store work with my specs by removing the two 1024 large fully connected layers i.e loss1/fc and loss2/fc (and the BN and relu that followed) . This removed the:
ERROR: Check failed: axis_index < num_axes() (2 vs. 2) axis 2 out of range for 2-D Blob with shape 64 1024 (65536).

Training seems to be going fine atm..
